icicle/src/hash/keccak/extern.cu

```diff
@@ -4,17 +4,44 @@
 #include "hash/hash.cuh"
 #include "hash/keccak/keccak.cuh"
 #include "keccak.cu"
+#include "../../merkle-tree/merkle.cu"
+#include "merkle-tree/merkle.cuh"
 
 namespace keccak {
   extern "C" cudaError_t
-  keccak256_cuda(uint8_t* input, int input_block_size, int number_of_blocks, uint8_t* output, KeccakConfig& config)
+  keccak256_cuda(uint8_t* input, int input_block_size, int number_of_blocks, uint8_t* output, HashConfig& config)
   {
-    return keccak_hash<512, 256>(input, input_block_size, number_of_blocks, output, config);
+    return Keccak(136).hash_many(input, (uint64_t*)output, number_of_blocks, input_block_size, 4, config);
```

Comment:
Do we want to make these hardcoded values global consts? We have them in a few places in the code and should probably make it a single place

Reply:
fixed in [f607239](https://github.com/ingonyama-zk/icicle/pull/556/commits/f607239e705c50391b72575809345f3ec90a2a65)